server/sonar-server/src/main/java/org/sonar/server/projectanalysis/ws/UpdateEventAction.java

```diff
@@ -95,12 +95,14 @@ public void handle(Request httpRequest, Response httpResponse) throws Exception
   }
 
   private UpdateEventResponse doHandle(UpdateEventRequest request) {
+    String name = request.getName();
+    checkArgument(name == null || name.length() <= 100, "The name must not be longer then 100 characters.");
```

Comment:
The form below is used elsewhere:
```
Event name length (%s) is longer than the maximum authorized (%s). '%s' was provided.
```

Reply:
I wonder if we really want to restrict event length to 100 characters for events that are not Version events

Reply:
Done.